test_cases/stdlib/check_unittest.py

```diff
@@ -114,4 +113,3 @@ def method() -> int:
 
 
 assert_type(TestXYZ.attr, int)
```

Comment:
```suggestion
assert_type(TestXYZ.attr, int)
assert_type(TestXYZ.method(), int)
```
Keep some of the intent of the test